homeassistant/components/tplink/sensor.py

```diff
@@ -34,23 +38,73 @@
 from .coordinator import TPLinkDataUpdateCoordinator
 from .entity import CoordinatedTPLinkEntity
 
+_LOGGER = logging.getLogger(__name__)
+
+
+AcceptedValueFnReturnValues = float | datetime.datetime | None
+
 
 @dataclass
 class TPLinkSensorEntityDescription(SensorEntityDescription):
     """Describes TPLink sensor entity."""
 
-    emeter_attr: str | None = None
+    attribute_name: str | None = None
```

Comment:
I think it's mandatory to define these with default values to avoid mypy's `error: Attributes without a default cannot follow attributes with one  [misc]`.

I'm open for suggestions, as it's not really nice as this will require checking these later for Noneness.

Reply:
You can instead use a "Mixin" with double inheritance.

Reply:
Great, thanks for the tip!